packages/wonder-blocks-dropdown/components/multi-select.test.js

```diff
@@ -132,5 +132,9 @@ describe("MultiSelect", () => {
         // All items are selected
         select.setProps({selectedValues: ["1", "2", "3"]});
         expect(opener.text()).toEqual("All students");
+
+        // implicitAllEnabled
+        select.setProps({implicitAllEnabled: true, selectedValues: []});
```

Comment:
Instead of adding these lines, could you please create a separate unit test checking this new behavior? Also, It would be great if you add comments to follow the `Arrange, Act, Assert` guidelines :)